### PR TITLE
🎨  switch to `debugpy`

### LINKED ISSUE #5343: Switch to `debugpy` everywhere
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Planned changes

This is a sub issue from https://github.com/ITISFoundation/osparc-issues/issues/675

### PR DESCRIPTION
**Pull request by bisgaard-itis**
## What do these changes do?
- Switch to using `debugpy` for debugging local deployment

## Related issue/s

- resolves #5343 

## Dev Checklist

- [x] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

### CODE REVIEW

**Review by pcrespov — APPROVED** [review]
Great! Thx so much!
Does the `.vscode/launch.template.json` need to be updated as well or works as is?

File: services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/core/settings.py [hunk 1]
```diff
@@ -101,10 +101,6 @@ class ApplicationSettings(BaseCustomSettings, MixinLoggingSettings):
         description="If set to True the application will boot into debug mode",
     )
 
-    DYNAMIC_SIDECAR_REMOTE_DEBUG_PORT: PortInt = Field(
```
Comment (GitHK) [inline services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/core/settings.py:104]: you can also remove `DYNAMIC_SIDECAR_HOST` form the settings

**Review by GitHK — APPROVED** [review]

**Review by sanderegg — APPROVED** [review]
very nice thanks!

**Review by matusdrobuliak66 — APPROVED** [review]

File: services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/core/settings.py (hunk 1 above)
Comment (bisgaard-itis) [reply services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/core/settings.py:104]: Done

File: services/docker-compose.local.yml [hunk 2]
```diff
@@ -48,40 +56,53 @@ services:
       - SC_BOOT_MODE=${SC_BOOT_MODE:-default}
       - SWARM_STACK_NAME=${SWARM_STACK_NAME:-simcore_local}
       - DYNAMIC_SIDECAR_PROMETHEUS_SERVICE_LABELS={}
+      - DIRECTOR_V2_REMOTE_DEBUGGING_PORT=3000
```
[the remark this answers is not in the document]
Comment (bisgaard-itis) [reply services/docker-compose.local.yml:59]: Yes, it is internal to the container but I still find it clearer to have the service name in front

**Pull request merged**